### PR TITLE
Dexter compiled macros

### PR DESCRIPTION
**Pull request by hjorthjort**
Macros `#dexterCode` and `#dexterCodeFA2` that represent the compiled code.
The code is copied verbatim, and some macros are added to make that possible.

### CODE REVIEW

File: tests/proofs/dexter/dexter-compiled.md [hunk 1]
```diff
@@ -0,0 +1,2928 @@
...
+                       | "SWAP"
+                       | "SOME"
+                       | "UNIT"
+                       | "PAIR"
+                       | "UNPAIR"
+                       | "CAR"
+                       | "CDR"
+                       | "CONS"
+                       | "SIZE"
+                       | "MEM"
```
Comment (ehildenb) [inline tests/proofs/dexter/dexter-compiled.md:2715]: Let's move this to a file in the repo root called `pretty-instructions.md` or something, in its own module, then just import that module here.

File: tests/proofs/dexter/dexter-spec.md [hunk 2]
```diff
@@ -5,5 +5,8 @@ module DEXTER-SPEC
   claim <k> now 0 => . ... </k>
         <mynow> #Timestamp(0) </mynow>
 
+  claim <k> (now 0 => .) ~> #dexterCode ... </k>
+        <mynow> #Timestamp(0) </mynow>
+
```
Comment (ehildenb) [inline tests/proofs/dexter/dexter-spec.md:10]: We can likely add tests for each macro type using `claim <k> PRETTY_DEXTER_CODE ; POP ; POP ; POP => . ... </k>`, which makes it so it's a no-op on the michelson semantics side.

File: tests/proofs/dexter/dexter-compiled.md (hunk 1 above)
Comment (hjorthjort) [reply tests/proofs/dexter/dexter-compiled.md]: Done

File: tests/proofs/dexter/dexter-spec.md (hunk 2 above)
Comment (hjorthjort) [reply tests/proofs/dexter/dexter-spec.md:10]: Done

**Review by sskeirik — APPROVED** [review]

**Pull request merged**